From f2e4ad355a92d447462ce46b6515153d274d4daf Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Thu, 2 Aug 2018 11:46:04 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3308: Reduce 25000uV for 816MHz and
 1008MHz

The latest Vmin test result on evb board is as the following.
/* KHz    uV */
1296000   1250000
1200000   1200000
1008000   1075000
816000    975000
600000    850000
408000    850000
Each frequency should add 50000uV margin.

Change-Id: Icbe99d2a4d88466381fd418fb74a8b8e9cac7323
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index c25797f69a48..61628a3013e9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -114,12 +114,12 @@
 		};
 		opp-816000000 {
 			opp-hz = /bits/ 64 <816000000>;
-			opp-microvolt = <1050000 1050000 1340000>;
+			opp-microvolt = <1025000 1025000 1340000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
-			opp-microvolt = <1150000 1150000 1340000>;
+			opp-microvolt = <1125000 1125000 1340000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-1200000000 {
-- 
2.35.3

